### DISCUSSION TITLE
Auto generated usernames?

### DISCUSSION THREAD

**Discussion by regionbibliotekvasterbotten** [mozilla/hubs#4057] [General Questions & Discussion] [answered]
Hi,
We are having a conference with Mozilla Hubs as the plattform at the moment and some of the participants reacted to some of the auto generated usernames. For example "Hottentot" and so on. It there a way to control which names Hubs are using for the auto generated usernames? Or another solution to this problem?

Sincerily  /Johan

**Comment by yalegria** [comment]
Users should be able to choose/specify a name for their avatar from the top left menu.

**Comment by brianpeiris** [comment, marked as answer]
Thanks for raising this, Johan. We've just released an update to hubs.mozilla.com to remove that particular name from the list. On hubs.mozilla.com, we don't give users a way to control that list, but if you were running your own servers with Hubs Cloud, you could deploy a customized client that included a list of your choice.

**Reply by regionbibliotekvasterbotten** [reply]
Thanks for the quick response and action!

Ah, we actually have our own servers with Hubs Cloud. Will look into a customized client!
